CodingTracker.Speedierone/CodingTracker.Speedierone/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Data.Sqlite;
+namespace CodingTracker;
+
+//come back to link to config file.
```

Comment:
TODO comments are fine, but please follow this format: // TODO - come back  etc.